.travis.yml

```diff
@@ -21,14 +21,15 @@ services: docker
 env:
   matrix:
   - INSTANCE=ubuntu-1404
-  - INSTANCE=ubuntu-1604
+  # - INSTANCE=ubuntu-1604
```

Comment:
i'm a chef noob.  But any reason not to also target 16.04 in the travis builds? 

I know you modded this line https://github.com/maraaaa/chef-rocketchat/blob/8445466185f47062674f3974a9b6a42b76836c93/spec/unit/recipes/default_spec.rb#L6
to:
> runner = ChefSpec::ServerRunner.new(platform: 'ubuntu', version: '14.04')

I think probably good idea to keep testing 16.04 also if at all possible.  Because I know my self if I drop in a new Ubuntu LTS box its going to be 16.04 and not 14.04.